src/Components/FilterSection/CitizenshipPopover.tsx

```diff
@@ -39,49 +41,124 @@ const CitizenshipPopover = ({
+      // if the citizenshipType is `green_card`, then set green_card and all the gc_options to true or false
+      // i.e. green_card and all the gc_options should be the same when citizenshipType is `green_card`
+      updatedCitizenshipFilterIsChecked = {
+        ...updatedCitizenshipFilterIsChecked,
+        gc_5plus: !isChecked,
+        gc_18plus_no5: !isChecked,
+        gc_under18_no5: !isChecked,
+        gc_under19_pregnant_no5: !isChecked,
+      };
+    }
     const typedUpdatedCitizenshipFilterIsChecked = Object.keys(updatedCitizenshipFilterIsChecked) as CitizenLabels[];
     const selectedCitizenshipFilters = typedUpdatedCitizenshipFilterIsChecked.filter((citizenshipType) => {
       return updatedCitizenshipFilterIsChecked[citizenshipType];
     });
 
-    updateFilter({
-      name: 'citizen',
-      filter: {
-        id: 1,
-        columnField: 'citizenship',
-        operatorValue: 'customCitizenshipOperator',
-        value: selectedCitizenshipFilters,
-      },
-    });
+    //update the MUI filter that is being passed to the citizenship column
+    if (hasAtLeastOneCitizenshipFilter(updatedCitizenshipFilterIsChecked)) {
+      updateFilter({
+        name: 'citizen',
+        filter: {
+          id: 1,
+          columnField: 'citizenship',
+          operatorValue: 'customCitizenshipOperator',
+          value: selectedCitizenshipFilters,
+        },
+      });
+    } else {
+      // set the citizenship filter back to the default
+      updateFilter({
+        name: 'citizen',
+        filter: {
+          id: 1,
+          columnField: 'citizenship',
+          operatorValue: 'customCitizenshipOperator',
+          value: ['citizen'],
+        },
+      });
+    }
 
+    //update citizenshipFilterIsChecked state
     setCitizenshipFilterIsChecked(updatedCitizenshipFilterIsChecked);
   };
 
   const typedCitizenshipFilterIsChecked = Object.keys(citizenshipFilterIsChecked) as CitizenLabels[];
-  const citizenshipCheckboxFilters = typedCitizenshipFilterIsChecked.map((citizenshipType) => {
-    return (
-      <FormControlLabel
-        key={citizenshipType}
-        className="popover"
-        label={citizenshipFilterFormControlLabels[citizenshipType]}
-        control={
-          <Checkbox
-            checked={citizenshipFilterIsChecked[citizenshipType]}
-            onChange={() => handleFilterSelect(citizenshipType)}
+
+  const renderCitizenshipFilters = (citizenshipFilters: Record<CitizenLabels, boolean>) => {
+    if (citizenshipFilters.green_card) {
+      const allCitizenshipCheckboxes = typedCitizenshipFilterIsChecked.map((citizenshipType) => {
+        //here we need to add an sx prop to indent them if they're the gc_filters
+        const isGreenCardSubCitizenshipType =
+          citizenshipType === 'gc_5plus' ||
+          citizenshipType === 'gc_18plus_no5' ||
+          citizenshipType === 'gc_under18_no5' ||
+          citizenshipType === 'gc_under19_pregnant_no5';
+
```

Comment:
Could you map over all of the categories just not render the green card categories if green card is not selected. That would enable you to remove the long if statement.